### PR TITLE
chore: release notes for 1.30.0

### PR DESCRIPTION
**Pull request by Ilia**

### CODE REVIEW

File: website/docs/docs/on-premises/releases/1.30.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+08 April 2024
+
+## Features
+
+* Add team-specific metric details via an endpoint /metrics/teams
+
+## Fixes
+
+* Fix bug where a user with environment:team:read permission received a 403 when performing a GET request for an individual environment
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.30.0.md:15]:
```suggestion
* Fix bug where a user with `environment:team:read` permission received a 403 when performing a GET request for an individual environment
```

File: website/docs/docs/on-premises/releases/1.30.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Ensure matrix and can-i-deploy use the latest revision of the pact when using bi-directional contracts and the pact for a consumer version is modified
+* Fix the bug where searching integrations in the dashboard would search it from the list of first 1000 integrations
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.30.0.md:17]:
```suggestion
* Fix bug where searching integrations in the dashboard would only search from the first 1000 integrations
```

File: website/docs/docs/on-premises/releases/1.30.0.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* Improved performance of the /admin/users endpoint
```
Comment (mefellows) [inline website/docs/docs/on-premises/releases/1.30.0.md:18]:
```suggestion
* Improved performance of the `/admin/users` endpoint
```

**Review by mefellows — APPROVED** [review]
Minor changes (posted as suggestions)

File: website/docs/docs/on-premises/releases/1.30.0.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: 1.30.0
+---
+
+## Release date
+
+08 April 2024
```
Comment (tuan-pham) [inline website/docs/docs/on-premises/releases/1.30.0.md:7]: Need to be on the release date (maybe today)

**Pull request merged**